src/timePicker/TimePicker.style.tsx

```diff
@@ -53,6 +64,10 @@ export const StyledTimePicker = styled.div`
     border-color: transparent;
   }
 
+  &[aria-disabled='true']:hover {
+    background-color: ${color.white};
+  }
```

Comment:
not sure to understand this rule?

Reply:
We had an issue when the field is disabled, we were still applying a gray background on hover. The text also being gray, we just saw a gray block

Reply:
It seems that this `disabled` prop does not implemented in the design system and that we don't use it on SPA side. I'll be in favor of removing this. WDYT?

Reply:
Do we have a disabled state for all form elements? For consistency's sake I'll like to have the same everywhere, but as we this TimePicker only appears alone in One Action screens I guess we can remove it. @gabrielagabriel WDYT?

Reply:
Its indeed an global issue on realign libraries. PIXAR doesn't have hover, focus and disabled states. 
I think we can treat with other components (when spec will be created)